### PR TITLE
Adding image support

### LINKED ISSUE #5: Add image support to ebooks
We've discussed a little how this might work - possibly image filename or filename + path in the front matter, then interpolated into the template...

### PR DESCRIPTION
**Pull request by sirkitree**
Resolves #5 

- new images dir within each ebook sample
- ensure we copy these in .eleventy.js
- new image for jponch-sample/chapter-1.md in front matter
- include the full path in chapter.liquid
image

### CODE REVIEW

**Comment by sirkitree** [comment]
This should probably be using your image shortcode stuff. i'll figure that out.

**Comment by sirkitree** [comment]
Actually, since that's more realted to the responsive stuff, I think we can leave that for #6

File: .eleventy.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const Image = require("@11ty/eleventy-img");
 const path = require("path");
+const fs = require("fs");
```
Comment (huntermacd) [inline .eleventy.js:3]: Is this needed?

**Review by huntermacd — COMMENTED** [review]
One question. Otherwise looks great! And yeah, we can make use of the shortcode in another PR.

File: .eleventy.js (hunk 1 above)
Comment (sirkitree) [reply .eleventy.js:3]: ah, nope it's not. I was having trouble getting it to copy the files and tried using fs to crawl the dir struct for the addPassthroughCopy but that didn't work and I neglected to remove this.

**Pull request merged**